.github/workflows/quick-test.yml

```diff
@@ -46,12 +46,32 @@ jobs:
         run: |
             export DEBIAN_FRONTEND=noninteractive
             sudo apt-get update
-            sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev clang-${{ matrix.clang }} libc++-${{ matrix.clang }}-dev libc++abi-${{ matrix.clang }}-dev
+            sudo apt-get install --no-upgrade -y build-essential git zlib1g-dev cmake libssl-dev clang-${{ matrix.clang }} libc++-${{ matrix.clang }}-dev libc++abi-${{ matrix.clang }}-dev
       - name: test
         run: |
             export CC=clang-${{ matrix.clang }}
             export CXX=clang++-${{ matrix.clang }}
             ${{ matrix.run-test }}
+  Linux-clang-sanitizers:
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        clang: [18]
+        config: [asan]
+    steps:
+      - uses: actions/checkout@v4
+      - name: install dependencies
+        # I observed 404s for some packages and added an `apt-get update`.
+        run: |
```

Comment:
not sure if there's a way to dedup this step here? @fhanau

Reply:
You could add `driver: bazel-asan` `run-test:  cd c++ && bazel test --verbose_failures --test_output=errors -k --config=asan //...` to `Linux-clang` above and then filter out clang versions other than 18. Might not be much shorter though

Reply:
So, I wanted to try yaml anchors, but apparently they are not supported on github actions: https://github.com/orgs/community/discussions/4501

Reply:
OK, I did some bash hackery. How about now? @fhanau